### ISSUE TITLE
Feature suggestion: Make voice directions time configurable

### ISSUE THREAD

**Issue by wonder75** [organicmaps/organicmaps#3989]
Only thing that bugs me constantly is the time voice directions are given. There is often not enough time left
to follow the directions in dense city traffic. Things like "keep right" or "turn left", etc. should be announced just a little bit earlier
for my driving style and the traffic conditions in my city. Would be very nice to be able to add a configurable amount of seconds to
the announcements to hear them earlier and make things more relaxed. 

Thanks a lot.
Great project.

**Comment by biodranik** [comment]
Let's improve it without adding complexity by using the additional setting. Can you please provide some coordinates and routes in OpenStreetMap.org where you always reproduce the same issue, and also describe to us an approximate car speed at that moment, so we can reproduce and test it?

**Comment by Shura0** [comment]
I agree with TS, but in other direction. Now OM announces turns too early
Example. I go with speed 70km/h, OM announces 'turn right in 600 meters' Ok, 600 meters may be reasonable in a city on multi-lanes roads to move to right lane in tight traffic. But next announce 'turn right' (means turn right here) will be 150 meters before real turn. 150 meters is too many for 'turn here' there may be other crossroad. It confuses me every time and I have to look at screen to make sure the turn is right
